lib/src/sep/0099/hotswap.dart

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 The Stellar Flutter SDK Authors. All rights reserved.
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:dio/dio.dart';
+import 'package:stellar_flutter_sdk/stellar_flutter_sdk.dart';
+
+class Hotswap {
+  Hotswap({
+    required this.destinationAddress,
+    required this.currentAsset,
+    required this.replacementAsset,
+    required this.minimumAmount,
+    required this.endpointUrl,
+    this.dio,
+  });
+  final String destinationAddress;
+  final String currentAsset;
+  final String replacementAsset;
+  final double minimumAmount;
+  final Uri endpointUrl;
+  Dio? dio;
+
+  /// Constructor
+  /// - Parameter url: Endpoint to be used for fetching the information from as well as sending the transaction to.
+  /// - Parameter dio: (Optional) the Dio client.
+
+  static Future<Hotswap> fromEndpointUrl(
```

Comment:
I think we should try to follow the appraoch in the rest of the SDK with a `fromDomain`.